net2cog/netcdf_convert.py

```diff
@@ -282,7 +282,15 @@ def netcdf_converter(
     if netcdf_file.endswith(('.nc', '.nc4', 'h5')):
         logger.info("Reading %s", basename(netcdf_file))
 
-        input_datatree = xr.open_datatree(netcdf_file)
+        time_coder = xr.coders.CFDatetimeCoder(use_cftime=False)
```

Comment:
Two quick things:

1) Could you explain on the PR here what the effect of the `CFDatetimeCoder` is compared to `decode_times=False`?
2) The `time_coder` variable is just used in the call to `open_datatree` - this would be simpler to just set `decode_times=xr.coders.CFDatetimeCoder(use_cftime=False)`. I don't think that declaring a variable in this instance is really buying you any additional readability.